feat(UserBundle): Add FrontendLoginLinkHandler and update README associate (#15)
roadiz/core-bundle-dev-app: lib/RoadizUserBundle/README.md

### Override login link URL

> A service was enable for generate a login-link  with a different **base-uri**,

Reviewer: ```suggestion
Roadiz User Bundle provides a custom `Symfony\Component\Security\Http\LoginLink\LoginLinkHandlerInterface` service to generate a login-link  with a different **base-uri**,
```

> for example if you are using a different domain for your frontend application.

Reviewer: ```suggestion
all you need is to register `RZ\Roadiz\UserBundle\Security\Security\FrontendLoginLinkHandler` service in your project with its mandatory arguments:
```

```yaml
# config/services.yaml
services:

>     App\Security\LoginLinkHandler:

Reviewer: Be careful, we instruct projects developer to use our FrontendLoginLinkHandler, not their anymore

```suggestion
    RZ\Roadiz\UserBundle\Security\Security\FrontendLoginLinkHandler:
```

    decorates: Symfony\Component\Security\Http\LoginLink\LoginLinkHandlerInterface
    arguments:

>         $decorated: '@App\Security\LoginLinkHandler.inner'

Reviewer: ```suggestion
        $decorated: '@RZ\Roadiz\UserBundle\Security\Security\FrontendLoginLinkHandler.inner'
```

        $frontendLoginCheckRoute: '%frontend_login_check_route%'

>         $frontendLoginLinkRequestRoutes :

Reviewer: ```suggestion
        $frontendLoginLinkRequestRoutes:
```

            - 'frontend_login_link_request_route'
            - 'another_login_link_request_route'
        $signatureHasher: '@security.authenticator.login_link_signature_hasher.api_login_link'
```

> Now for each `$frontendLoginLinkRequestRoutes` login_link redirect to `$frontendLoginCheckRoute`

Reviewer: ```suggestion
Now for each `$frontendLoginLinkRequestRoutes` login_link will be generated using `$frontendLoginCheckRoute` base URL
```
